src/Shared/Recruit.Vacancies.Client/Application/EventHandlers/NotifyOnVacancyActionsEventHandler.cs

```diff
@@ -5,41 +5,54 @@
 using Microsoft.Extensions.Logging;
 using System.Threading;
 using System.Threading.Tasks;
+using Communication.Types;
+using Esfa.Recruit.Vacancies.Client.Application.Communications;
 using Esfa.Recruit.Vacancies.Client.Domain.Repositories;
 using Esfa.Recruit.Vacancies.Client.Domain.Entities;
+using Esfa.Recruit.Vacancies.Client.Infrastructure.StorageQueue;
 
 namespace Esfa.Recruit.Vacancies.Client.Application.EventHandlers
 {
     public class NotifyOnVacancyActionsEventHandler : INotificationHandler<VacancyClosedEvent>,
```

Comment:
The handling stuff you have addressed in this class should go in the VacancyClosedEventHandler, not this one. Or create a separate out of process event handler which initiates the comm sending stuff